integration/mocha-tests/configs/BrowserCompatibilityBrowserConfig.js

```diff
@@ -0,0 +1,69 @@
+const browserCompatibilityConfig = [
```

Comment:
How do we override this with each test? Some tests only work on specific browsers.